ui/redux/actions/collections.js

```diff
@@ -333,18 +334,23 @@ export const doFetchItemsInCollection = (options: { collectionId: string, pageSi
   return doFetchItemsInCollections(newOptions, cb);
 };
 
-export const doCollectionEdit = (collectionId: string, params: CollectionEditParams) => async (
+export const doCollectionEdit = (collectionId: string, params: CollectionEditParams) => (
```

Comment:
- [ ] Anonymous list becomes non-anonymous if updated

I made sure to make the channel-selector in the page to be anonymous, but the API call still included `channel_id`.  I think it is active the default channel ID (same as the avatar)